### ISSUE TITLE
VS Code editor not recognising multiple script type tags in HTML

### ISSUE THREAD

**Issue by GuitarHero1969** [microsoft/vscode#99224]
- VSCode Version: 1.45.1
- OS Version: Windows_NT x64 10.0.18363

Steps to Reproduce:

0. Enter the JSON snippet in my comment into a local folder and point VS Code extensions to that.
1. Open up VS Code (a blank untitled file editor window is opened)
2. Click on the 'Plain Text' link in the bottom status bar and select HTML from drop-down list
3. Type a single exclamation mark and 'Emmet Abbreviation' allows html5 boilerplate to be entered
4. try entering script tag in the head element

Does this issue occur when all extensions are disabled?: Yes/No

**Comment by GuitarHero1969** [comment]
I am trying to emmet auto-complete to insert script tags in the html head, but it will only recognise the default: <script>|</script>.

I have added the following in a global local file that I have set extension path in VSCode to.
{
  "html": {
    "snippets": {   
	  "script": "<script type='text/javascript' src='${1}'></script>",
	  "script": "<script type='text/paperscript' src='${1}'></script>",
	  "script": "<script type='module'></script>"	  
    }
  }
}

But in a new HTML file in VS Code, only the first script option is available and then it doesn't insert the type attribute (which I realise isn't strictly necessary nowadays in html5).

I tried putting the above snippet into AppData | Roaming | Code | User | snippets | html.json, but it complained about multiple, identical keys.
